From 8be77c5de65442b331a28d63802c7a3b94a06c5a Mon Sep 17 00:00:00 2001
From: Todd Zullinger <tmz@pobox.com>
Date: Fri, 3 Nov 2023 10:17:51 -0400
Subject: [PATCH 2364/2871] RelNotes: improve wording of credential helper
 notes

Offer a slightly more verbose description of the issue fixed by
7144dee3ec (credential/libsecret: erase matching creds only, 2023-07-26)
and cb626f8e5c (credential/wincred: erase matching creds only,
2023-07-26).

Signed-off-by: Todd Zullinger <tmz@pobox.com>
Signed-off-by: Junio C Hamano <gitster@pobox.com>
---
 Documentation/RelNotes/2.43.0.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Documentation/RelNotes/2.43.0.txt b/Documentation/RelNotes/2.43.0.txt
index 3b4c0c804e..836fcb5433 100644
--- a/Documentation/RelNotes/2.43.0.txt
+++ b/Documentation/RelNotes/2.43.0.txt
@@ -37,8 +37,9 @@ UI, Workflows & Features
    oath token and password expiration data) are stored in libsecret
    keyrings has been rethought.
 
- * Update two credential helpers to correctly match which credential
-   to erase; they dropped not the ones with stale password.
+ * Update the libsecret and wincred credential helpers to correctly
+   match which credential to erase; they erased the wrong entry in
+   some cases.
 
  * Git GUI updates.
 
-- 
2.39.3 (Apple Git-145)

